lib/__tests__/client.test.js

```diff
@@ -1,5 +1,6 @@
 const APIClient = require('../client');
 const constants = require('../constants');
+const { bills } = require('../apis');
```

Comment:
I think we should support node v4. It is necessary for services like *AWS Lambda*.
The 'destructuring assignment' doesn't supported.

Please refer to [this commit](https://github.com/yongdamsh/enertalk-api-client/commit/17c4dc4170ff68a8f625403df32a781209a7d51c).